src/SpeechRecognition.js

```diff
@@ -71,8 +71,12 @@ export default function SpeechRecognition(options) {
         listening = false
         if (pauseAfterDisconnect) {
           this.setState({ listening })
-        } else if (recognition && recognition.continuous) {
-          this.startListening()
+        } else if (recognition) {
+          if(recognition.continuous) {
```

Comment:
Minor styling tweak: add space between `if` and `(recognition.continuous) {`.

Same below: `}else{` to `} else {`